README.md

```diff
@@ -7,7 +7,10 @@ __`redis-rails`__ provides a full set of stores (*Cache*, *Session*, *HTTP Cache
 Rails 5.2.0 [includes a Redis cache store out of the
 box](https://github.com/rails/rails/pull/31134), so you don't really
 need this gem anymore if you just need to store the fragment cache in
-Redis. Maintenance on the [redis-activesupport](https://github.com/redis-store/redis-activesupport)
+Redis. If you need a session cache, you may use
```

Comment:
It's not really a "session cache", it's "session storage", and there's more info about it right here: https://github.com/redis-store/redis-rails#session-storage

Can you move the `redis-actionpack` link to that section a little further down?

Reply:
> move the `redis-actionpack` link to that section a little further down

Done, but I fear that lazy people such as myself might not jump down there once we see the general "not needed in Rails anymore" section.

Reply:
maybe you could append the last line like:

```diff
- We are still actively maintaining all other gems in the redis-store family.
+ We are still actively maintaining all other gems in the redis-store family, such as [redis-actionpack](https://github.com/redis-store/redis-actionpack) for session management, and [redis-rack-cache](https://github.com/redis-store/redis-rack-cache) for HTTP cache storage.

Reply:
Yeah, that seems better!